stable_baselines3/common/vec_env/base_vec_env.py

```diff
@@ -162,35 +168,44 @@ def step(self, actions: np.ndarray) -> VecEnvStepReturn:
         self.step_async(actions)
         return self.step_wait()
 
-    def get_images(self) -> Sequence[np.ndarray]:
+    def get_images(self) -> Sequence[Optional[np.ndarray]]:
         """
-        Return RGB images from each environment
+        Return Render output from each environment
         """
         raise NotImplementedError
 
-    def render(self, mode: str = "human") -> Optional[np.ndarray]:
+    def render(self, mode: Optional[str] = None) -> Optional[np.ndarray]:
         """
         Gym environment rendering
 
         :param mode: the rendering type
         """
-        try:
-            imgs = self.get_images()
-        except NotImplementedError:
-            warnings.warn(f"Render not defined for {self}")
-            return
 
-        # Create a big image by tiling images from subprocesses
-        bigimg = tile_images(imgs)
-        if mode == "human":
-            import cv2  # pytype:disable=import-error
+        if mode and self.render_mode != mode:
+            raise ValueError(
```

Comment:
should be a warning I think (and return right after).
That way it does not crash.

Reply:
btw, I think we should support `mode="human"` when `render_mode="rgb_array"` because it is how it works with subprocess env anyway.